1.9/storage/persistent-volume.md

```diff
@@ -142,7 +140,7 @@ However, if another framework does not respect the presence of labels and the se
 
 ### The Mesos Sandbox
 
-The temporary Mesos sandbox is still the target for the `stdout` and `stderr` logs. To view these logs, go to the Marathon pane of the DC/OS web interface.
+The temporary Mesos sandbox is still the target for the `stdout` and `stderr` logs. To view these logs, go to the Marathon pane of the DC/OS GUI.
```

Comment:
And finally `GUI` vs `UI` vs `web interface`.

To me personally, all are fine even in combination. Feel free to ignore these comments.